contracts/test/NFT.sol

```diff
@@ -0,0 +1,31 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+import "hardhat/console.sol";
+import "@openzeppelin/contracts/token/ERC721/extensions/ERC721URIStorage.sol";
+import "@openzeppelin/contracts/utils/Counters.sol";
+
+contract NFT is ERC721URIStorage {
+    using Counters for Counters.Counter;
+    Counters.Counter private _tokenIds;
+
+    mapping(uint256 => bool) private _communityMemberShip;
+
+    constructor() ERC721("Henkaku v0.1", "henkaku") {}
+
+    function isCommunityMember(uint256 _tokenId) public view returns(bool) {
+      return _communityMemberShip[_tokenId];
+    }
```

Comment:
If you pass address instead of tokenId (or you create different function to map the reverse relationship), then you can avoid the loop at https://github.com/henkaku-center/quest-management/pull/1/files#diff-f853073c5ff70d7a90b70de8742604d1eff1b62505039d2e48c3edc0d951545eR87

Reply:
yeah, but already actual NFT is in production.
so, that was the reason I said in discord I wanna make V2 membership NFT.